test/shm_size_negative.bats

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bats
```

Comment:
can these be in the same file, but different tests?

Reply:
Oh, they probably can be lol